### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by haramkim-1**
Refactor sql server engine and client library into separate folders

### CODE REVIEW

**Review by abhinavp99 — APPROVED** [review]
all looks good

File: server/src/DBExecutorServer.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import * as express from 'express';
```
Comment (T40TDS) [inline server/src/DBExecutorServer.ts:1]: Was this file intended to be in this diff?

Comment (T40TDS) [reply server/src/DBExecutorServer.ts:1]: Nevermind

File: server/src/DBExecutorServer.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import * as http from 'http';
```
Comment (T40TDS) [inline server/src/DBExecutorServer.ts:2]: I would add some documentation as to how this is meant to be used and works

**Review by T40TDS — APPROVED** [review]
